packages/cf-component-checkbox/src/CheckboxGroup.js

```diff
@@ -34,7 +34,7 @@ class CheckboxGroup extends React.Component {
             name={option.name}
             value={option.value}
             checked={includes(this.props.values, option.value)}
-            onChange={val => this.handleChange(option.value, val)}/>;
+            onChange={e => this.handleChange(option.value, e.target.checked)}/>;
```

Comment:
@jwineman You asked me because of this? If you are doing micro-optimizations, using immutable data structures and pure components that implement `shouldComponentUpdate()`, this is not a good solution because it [creates a new function with a new identity](https://medium.com/@esamatti/react-js-pure-render-performance-anti-pattern-fb88c101332f#.ml95bv4bt), so the `render()` of Checkbox is called every single time. But we don't do that, so it doesn't really matter and you need to pass that option param somehow.

Anyway, is there some reason why we don't use functional components?